code/src/containers/forms/product-form/ProductForm.constants.ts

```diff
@@ -0,0 +1,10 @@
+export const productLocales = [
+  { key: "en", translationKey: "language.full.english" },
+  { key: "uk", translationKey: "language.full.ukrainian" }
+] as const;
```

Comment:
This object is disconnected from `Locale` type. Try to make typing more strict, for example as follows:

```tsx
type ProductLocale = {
    key: Locale;
    translationKey: string;
}

export const productLocales: ProductLocale[] = [
  { key: "en", translationKey: "language.full.english" },
  { key: "uk", translationKey: "language.full.ukrainian" }
] as const;
```

Reply:
Thank you, fixed